qdrant-landing/content/documentation/guides/distributed_deployment.md

```diff
@@ -171,16 +169,14 @@ client.createCollection("{collection_name}", {
 });
 ```
 
-We recommend selecting the number of shards as a factor of the number of nodes you are currently running in your cluster.
+We recommend setting the number of shards to be a multiple of the number of nodes you are currently running in your cluster.
 For example, if you have 3 nodes, 6 shards could be a good option.
 
-### Cluster scaling
+Shards are evenly distributed across all existing nodes when a collection is first created, but Qdrant does not automatically rebalance shards if your cluster size changes (since this is an expensive operation on large clusters). See the next section for how to move shards after scaling operations.
 
-If you want to extend your cluster with new nodes or some nodes become slower than the others, it might be helpful to re-balance the shard distribution in the cluster.
+### Moving shards
 
-*As of v0.9.0:* Qdrant allows moving shards between nodes in the cluster and removing nodes from the cluster.
-
-This functionality unlocks the ability to dynamically scale the cluster size without downtime.
+*As of v0.9.0:* Qdrant allows moving shards between nodes in the cluster and removing nodes from the cluster. This functionality unlocks the ability to dynamically scale the cluster size without downtime. It also allows you to upgrade or migrate nodes without downtime.
```

Comment:
We format it this way in other places. I know you didn't change this, but I guess we can make it consistent as part of this PR as well now.

```suggestion
*Available as of v0.9.0

Qdrant allows moving shards between nodes in the cluster and removing nodes from the cluster. This functionality unlocks the ability to dynamically scale the cluster size without downtime. It also allows you to upgrade or migrate nodes without downtime.
```